src/posix/platform/udp.cpp

```diff
@@ -149,7 +153,7 @@ static otError transmitPacket(int aFd, uint8_t *aPayload, uint16_t aLength, cons
 #ifdef __APPLE__
     msg.msg_controllen = static_cast<socklen_t>(controlLength);
 #else
-    msg.msg_controllen = controlLength;
+    msg.msg_controllen           = controlLength;
```

Comment:
align?

Reply:
it's already the result of clang-format.